bedrock/firefox/templates/firefox/channel/base.html

```diff
@@ -1,71 +1,82 @@
-          <li class="nav-ios"><a href="{{ url('firefox.channel.ios') }}" data-link-type="nav" data-link-name="iOS">{{ _('iOS') }}</a></li>
+  <nav class="c-sub-navigation">
+    <div class="mzp-l-content">
+      <div class="c-sub-navigation-content">
+        <h2 class="c-sub-navigation-title"></h2>
+        <ul class="c-sub-navigation-list">
+          <li class="c-sub-navigation-item"><a href="{{ url('firefox.channel.desktop') }}" data-link-type="nav"
+              data-link-position="subnav" data-link-name="Desktop">{{ _('Desktop') }}</a></li>
+          <li class="c-sub-navigation-item"><a href="{{ url('firefox.channel.android') }}" data-link-type="nav"
+              data-link-position="subnav" data-link-name="Android">{{ _('Android') }}</a></li>
+          <li class="c-sub-navigation-item"><a href="{{ url('firefox.channel.ios') }}" data-link-type="nav"
+              data-link-position="subnav" data-link-name="iOS">{{ _('iOS') }}</a></li>
         </ul>
       </div>
-    </nav>
-  </header>
-  {% block channels %}{% endblock %}
-  <div id="help-out">
-    <div class="container">
-      <section class="bugzilla">
-        <h3>{{ _('See something that isn’t working? Let us know.') }}</h3>
-        <ul>
-          <li><a rel="external" class="more" href="https://bugzilla.mozilla.org/enter_bug.cgi">{{ _('File a bug now') }}</a></li>
-          <li><a rel="external" class="more" href="https://developer.mozilla.org/docs/Mozilla/QA/Bug_writing_guidelines">{{ _('Tips for filing a bug') }}</a>
-        </ul>
-      </section>
     </div>
+  </nav>
+
+  {% call hero(
+    title=_('Take a browse on the wild side.'),
+    desc=_('Be among the first to explore future releases of Firefox for desktop, Android and iOS.'),
+    class='mzp-t-firefox mzp-t-product-firefox',
+    include_cta=False,
+    heading_level=1) %}
+  {% endcall %}
+
+  <div class="mzp-l-content mzp-l-narrow">
+    {% block channels %}{% endblock %}
   </div>
-  <aside id="newsletter-subscribe">
-    <div class="container">
-      {{ email_newsletter_form(title=_('<span>Firefox</span> + You'), subtitle=_('Get Firefox tips, tricks, news and more'), button_class='button-light') }}
+
+  {% call picto_card(
+    title=_('See something that isn’t working? Let us know.'),
+    heading_level=2,
+    class='bugzilla',
+    custom_desc=True) %}
+
+  <ul>
+    <li><a class="mzp-c-cta-link" href="https://bugzilla.mozilla.org/enter_bug.cgi">{{ _('File a bug now') }}</a></li>
+    <li><a class="mzp-c-cta-link" href="https://developer.mozilla.org/docs/Mozilla/QA/Bug_writing_guidelines">{{ _('Tips for filing a bug') }}</a></li>
+  </ul>
+  {% endcall %}
+
+  <aside class="mzp-l-content">
+    <div class="mzp-c-newsletter">
+      <div class="mzp-c-newsletter-image">
+        {{ high_res_img('img/home/2018/newsletter-graphic.png', {'alt': ''}) }}
```

Comment:
I think the comment about the Mozilla branded signup is related to this graphic. A Firefox branded image might work better?

Reply:
I spoke with Cristina & we are going to reuse the styles from the newsletter on  firefox/developer for the time-being. 

![Screen Shot 2019-12-05 at 10 02 22 AM](https://user-images.githubusercontent.com/30483988/70261068-85504000-1746-11ea-9f49-8d2e7eaac10e.png)